lib/fathead/python_six_library/trigger_words.txt

```diff
@@ -0,0 +1,100 @@
+six PY2 
```

Comment:
@iamrajhans this is incorrect. the trigger words list should be a short list of words/phrases we expect someone to include before or after any of the Fathead article titles in the output.txt.

For example we have an article title `add_metaclass`. This means if someone searches "add_metaclass" we will should show the result from the output.txt for that article title.

It's likely that a user will search "python six add_metaclass", in this case "python six" is our trigger word, so it should be added to `trigger_words.txt`.

the `trigger_words.txt` should look like so:

```
six
six library
python six
python six library
```

that way, if any one of those phrase is found at the start or end of a query, it will tell us to check if the remainder matches an article title, and if so, we'll show the appropriate result.